backends/bmv2/psa_switch/psaSwitch.h

```diff
@@ -39,7 +39,9 @@ class PsaSwitchExpressionConverter : public ExpressionConverter {
             auto jsn = new Util::JsonObject();
             jsn->emplace("name", param->toString());
             jsn->emplace("type", "hexstr");
-            auto bitwidth = param->type->width_bits();
+            // FIXME -- how big is a PSA_CounterType_t?  Being an enum type, we can't
```

Comment:
Setting the target-specific enum width could be implemented as part of the compiler configuration class?

Reply:
Probably, though with this code as is, it just uses a size of 0, which results in generating unsized integer constants in the json.  It is not clear why this code is messing around with sizes -- it ends up just printing `"0"`, `"1"`, or `"2"` for the enum constant.  A lot of extraneous work to do nothing.